GCR installs it's own GLib log handler, which writes all messages to syslog.
In GLib, log handlers are responsible for filtering messages by their log level.
However, GCR's log handler does not filter any messages, thus also writing
debug messages to syslog, which quickly pollutes the authlog.

The offending log handler has been removed in GCR-4.

See: https://gitlab.gnome.org/GNOME/gcr/-/work_items/78

diff -upr gcr-3.41.2.orig/ui/gcr-prompter-tool.c gcr-3.41.2/ui/gcr-prompter-tool.c
--- gcr-3.41.2.orig/ui/gcr-prompter-tool.c	2026-06-13 14:12:00.465597685 +0200
+++ gcr-3.41.2/ui/gcr-prompter-tool.c	2026-06-13 14:12:34.159485229 +0200
@@ -154,8 +154,8 @@ log_handler (const gchar *log_domain,
 		level = LOG_INFO;
 		break;
 	case G_LOG_LEVEL_DEBUG:
-		level = LOG_DEBUG;
-		break;
+		// Do not log debug messages to syslog.
+		return;
 	default:
 		level = LOG_ERR;
 		break;
